charts/timed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.9.1
+version: 0.9.2
```

I would possibly bump the minor version of the Helm chart when you're updating the major version of the application being deployed by it.
```suggestion
version: 0.10.0
```

---

So it should be 0.11.0 then, right? The previous chart version doesn't show correctly as it should be 0.10.0